server/models/handlers/workerTimeslot.js

```diff
@@ -10,6 +10,18 @@ async function addWorkerTimeslot(slotId, schoolId, workerId, status, date) {
         return false;
     }
 };
+
+async function updatingTheWorkerAvailability(status){
+    try {
+        await timeslotModel.insertWorkerTimeslot(status);
+        return true;
+    } catch (error) {
+        console.log('Error occurred in updatingTheWorkerAvailability method: ', error);
+        return false;
+    }
```

Comment:
Hi Simran,

This try-catch block checks the insertWorkerTimeslot method, which has been previously tested. I would recommend that you place the try-catch method before the `updatingTheWorkerAvailability(status)` so that this new method is tested.

Reply:
Hi Nancy! 

Once again great catch! I will look into this comment and do some testing to ensure that the try-catch method is before the status. Thank you for your time! 

Simran

Reply:
Sounds good! As long as there is 100% test coverage, you should be good to go!